data/sql/updates/pending_db_world/rev_1613178860602736100.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1613178860602736100');
+
+/* Add missing emotes for Maybell Maclure when offering quest Young Lovers=id106
+   Source: https://www.youtube.com/watch?v=Y6XYsgc2epI
+*/
+
+DELETE FROM `quest_details` WHERE `ID` = 106;
+INSERT INTO `quest_details` (`ID`, `Emote1`, `Emote2`, `EmoteDelay2`, `VerifiedBuild`) VALUES ('106', '18', '20', '1', '12340');
```

Comment:
not critical but those values are numbers so there is no need to add them as strings

Reply:
better? 😃